admin/src/components/VendorTable.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState, useEffect } from "react";
+import { Button, Table } from "react-bootstrap";
+import "bootstrap/dist/css/bootstrap.min.css";
+import { fetchVendors } from "./Service";
+
+const VendorTable = () => {
+  const [allVendors, setAllVendors] = useState([]);
+
+  const getAllVendors = async () => {
+    const result = await fetchVendors();
+    setAllVendors(result);
+  };
+
+  useEffect(() => {
+    getAllVendors();
+  }, []);
+
+  return (
+    <div>
+      <div align="center">
```

Comment:
Ilyen attribútum szerintem már nincs is HTML5-ben vagy deprecated. Viszont van Bootstrap class rá: text-center